src/lib/WIFI/devWIFI.cpp

```diff
@@ -796,6 +801,238 @@ static void WebUploadForceUpdateHandler(AsyncWebServerRequest *request) {
   }
 }
 
+#if defined(RADIO_LR1121)
+static size_t expectedFilesize;
+static SX12XX_Radio_Number_t updatingRadio;
+extern LR1121Hal hal;
+static uint8_t packet[256];
+static size_t left_over;
+
+static void writeLR1121Bytes(uint8_t *data, uint32_t sector_size) {
```

Comment:
Is this not actually `sector_size`? It appears to be the size of the uploaded chunk and not the sector size